src/js-ipfs-api/index.test.js

```diff
@@ -1,14 +1,14 @@
 /* global it, expect, jest */
 const tryApi = require('./index.js')
 const { URL } = require('url')
-const IpfsApi = require('ipfs-http-client')
+const ipfsClient = require('ipfs-http-client')
 
 it('Should use the apiAddress (implicit http)', async (done) => {
   const opts = {
     apiAddress: '/ip4/1.1.1.1/tcp/1111',
     defaultApiAddress: '/ip4/127.0.0.1/tcp/5001',
     location: new URL('http://localhost:5001'),
-    IpfsApi: IpfsApi,
+    ipfsClient: ipfsClient,
```

Comment:
When the name is the same on both sides, you can just write it once (`foo,` instead of `foo: foo,`)

Reply:
Thanks!!!